src/Applications/LBAF.py

```diff
@@ -414,13 +414,8 @@ def get_output_file_stem(params, n_ranks):
         a_min_max = []
 
     # Instantiate runtime
-    rt = Runtime(
-        phase,
-        params.work_model,
-        params.criterion,
-        params.order_strategy,
-        a_min_max,
-        lgr)
+    rt = Runtime(phase, params.work_model, params.criterion, params.order_strategy, a_min_max,
```

Comment:
Please restore one parameter per line for legibility

Reply:
done